relay/relay.go

```diff
@@ -199,8 +215,9 @@ func (c *Client) recLoop() {
 		_, packet, err := c.conn.ReadMessage()
 		if err != nil {
 			select {
-			case <-c.done:
-				continue
+			case <-c.stopRead:
+				// dont send error if
+				// stop was requested
```

Comment:
Is there no select fall-through in Go?

Reply:
There's fallthrough for switch statements but not selects